apps/aehttpserver/lib/aehttpserver/web/controllers/balance_controller.ex

```diff
@@ -2,14 +2,14 @@ defmodule Aehttpserver.Web.BalanceController do
   use Aehttpserver.Web, :controller
 
   alias Aecore.Chain.Worker, as: Chain
-  alias Aewallet.Encoding
+  alias Aeutil.Bits
 
   def show(conn, params) do
-    case Encoding.decode(params["account"]) do
+    case Bits.decode58c(params["account"]) do
```

Comment:
Use the `Account.base58_decode` function to decode account hashes

Reply:
There is no such function